src/standard.rs

```diff
@@ -77,12 +78,23 @@ pub struct EncryptedDocument {
 #[derive(Debug, Clone)]
 pub struct EncryptedDocuments(pub HashMap<DocumentId, EncryptedDocument>);
 custom_newtype!(EncryptedDocuments, HashMap<DocumentId, EncryptedDocument>);
+#[derive(Debug, Clone)]
 pub struct PlaintextDocuments(pub HashMap<DocumentId, PlaintextDocument>);
 custom_newtype!(PlaintextDocuments, HashMap<DocumentId, PlaintextDocument>);
 
 create_batch_result_struct!(RekeyEdeksBatchResult, EdekWithKeyIdHeader, DocumentId);
-create_batch_result_struct!(StandardEncryptBatchResult, EncryptedDocument, DocumentId);
-create_batch_result_struct!(StandardDecryptBatchResult, PlaintextDocument, DocumentId);
+create_batch_result_struct_using_newtype!(
+    StandardEncryptBatchResult,
+    EncryptedDocument,
+    DocumentId,
+    EncryptedDocuments
+);
+create_batch_result_struct_using_newtype!(
```

Comment:
Do any of these changes to wrapper types affect the public API? Or do they just become aliases?

Reply:
I think in Kotlin and Python the wrapper types are erased/comments/type aliases? For the Java they would've changed it if there'd been a release.

Reply:
Case in point on this, I didn't need to make any changes to the Python or Kotlin tests, which would have been affected by this. So we're good.